auth-service/src/main/resources/templates/thymeleaf/fragments/template.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
 <html class="govuk-template " lang="en" xmlns:th="http://www.thymeleaf.org">
-
+<!--/*@thymesVar id="isProduction" type="java.lang.Boolean" */-->
 <head th:fragment="head (title)">
     <meta charset="utf-8">
     <title th:text="${title}">Delta</title>
     <meta content="width=device-width, initial-scale=1, viewport-fit=cover" name="viewport">
     <meta content="#0b0c0c" name="theme-color">
     <meta content="IE=edge" http-equiv="X-UA-Compatible">
+    <meta th:unless="${isProduction}" name="robots" content="noindex">
```

Comment:
I think you're correct to only do this for non-production because evidently people use google to find the site, which is fine.

But you also need to add this to the Delta website because there are public pages.